sportsreference/ncaaf/player.py

```diff
@@ -501,10 +507,41 @@ def extra_points_made(self):
         """
         return self._extra_points_made
 
+    @_int_property_decorator
+    def extra_points_attempted(self):
+        """
+        Returns an ``int`` of the total number of extra points the player
+        attempted.
+        """
+        return self._extra_points_attempted
+
+    @_float_property_decorator
+    def extra_point_percentage(self):
+        """
+        Returns a ``float`` of the percentage of extra points the player made.
+        """
+        return self._extra_point_percentage
+
     @_int_property_decorator
     def field_goals_made(self):
         """
         Returns an ``int`` of the total number of field goals the player made
         from any distance.
         """
         return self._field_goals_made
+
+    @_int_property_decorator
+    def field_goals_attempted(self):
+        """
+        Returns an ``int`` of the total number of field goals the player
+        attempted from any distance.
+        """
+        return self._field_goals_attempted
+
+    @_float_property_decorator
+    def field_goal_percentage(self):
+        """
+        Returns a ``float`` of the percentage (0.0 to 100.0) of field goals 
```

Comment:
Also, looks like there's an extra space at the end after "goals". Sorry for not catching this earlier!

Reply:
done!